### ISSUE TITLE
Val Acc is very low

### ISSUE THREAD

**Issue by muyuuuu** [tjmoon0104/pytorch-tiny-imagenet#4]
I download tiny imagenet in kaggle: https://www.kaggle.com/akash2sharma/tiny-imagenet

Run your code without any modificatoin, why Acc is very slow and hovering around 0.0053

Where did you download your dataset?

**Comment by henry-tujia** [comment]
same here

**Comment by THUYimingLi** [comment]
same here

**Comment by Pranavug** [comment]
Was anyone able to solve this? Still facing the same issue

@tjmoon0104 @henry-tujia @THUYimingLi @muyuuuu

**Comment by josueortc** [comment]
You might wanna take a look at this other repo: https://github.com/rcamino/pytorch-notebooks/blob/master/Train%20Torchvision%20Models%20with%20Tiny%20ImageNet-200.ipynb. They do some reformatting of the validation set labels, which I think it's the problem with the low accuracy here.

**Comment by tjmoon0104** [comment]
Try the new [`prepare_dataset.py`](https://github.com/tjmoon0104/pytorch-tiny-imagenet/blob/master/prepare_dataset.py) to prepare dataset.
